Xamarin.Forms.Platform.Android/Shapes/PolylineRenderer.cs

```diff
@@ -24,6 +25,9 @@ protected override void OnElementChanged(ElementChangedEventArgs<Polyline> args)
 
             if (args.NewElement != null)
             {
+                var points = args.NewElement.Points;
+                points.CollectionChanged += OnCollectionChanged;
```

Comment:
aren't you supposed to unhook from the old element ?

Reply:
and what if Points collection is fully replaced ?

Reply:
Both of your points are handled in the sample fix I provided in my original issue: https://github.com/xamarin/Xamarin.Forms/issues/11563#issuecomment-664039739

https://github.com/xamarin/Xamarin.Forms/pull/11576 seems to suffer from the same shortcomings.